src/design-pages/passkey_authentication.rst

```diff
@@ -0,0 +1,318 @@
+Passkey - Authentication in a centralized environment
+=======================================================
+
+Related ticket(s):
+------------------
+ * https://github.com/SSSD/sssd/issues/6228
+
+Problem statement
+-----------------
+The use of new tools to authenticate users, such as 2FA, U2F and FIDO2, is
+becoming increasingly popular and currently SSSD doesn’t provide any way to use
+the latter. This diminishes the value provided by SSSD in some environments
+like big companies and governments, where the usage of these authentication
+mechanisms is becoming a common pattern.
+
+SSSD should provide a way to authenticate users that use a passkey. This would
+add more value by attracting the types of users mentioned in the previous
+paragraph.
+
+For the purpose of this feature, passkey is a FIDO2 compatible device supported
+by the libfido2 library.
+
+Use cases
+---------
+As a user, I want to walk to any system on a network and log in (desktop and/or
+terminal) using a passkey device connected to the system. The user credentials
+will be stored in a central management system like FreeIPA or Active Directory.
```

Comment:
ATM We only support local login over SSSD+FIDO. We don't support SSH+SSSD+FIDO, right? Therefore "I want to walk to any system on a network" is not part of this design page.

Reply:
Right, we only support SSSD+FIDO. The part of the phrase that you are quoting is exactly the one that specifies that you need to be physically near the system to perform the authentication, you can't do it remotely by SSH or any other means.

Reply:
Please, rephrase it to make it more clear. To me, "walk to any system on a network" means to ssh to any system on a network.

Reply:
I think you should drop 'any system on a network'  and replace it with something like 'a system enrolled into a centralized identity management solution', this would be more clear.

Reply:
I rephrased it to take into account your feedback